--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,7 +24,7 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools>=77", "Cython", "numpy", "mkl-devel"]
+requires = ["setuptools", "Cython", "numpy", "mkl-devel"]
 [project]
 authors = [
   {name = "Intel Corporation", email = "scripting@intel.com"}
@@ -54,7 +54,7 @@ classifiers = [
 dependencies = ["numpy>=1.26.4", "mkl-service"]
 description = "MKL-based FFT transforms for NumPy arrays"
 dynamic = ["version"]
 keywords = ["DFTI", "FFT", "Fourier", "MKL"]
-license = "BSD-3-Clause"
+license = { file = "LICENSE.txt" }
 name = "mkl_fft"
 readme = {file = "README.md", content-type = "text/markdown"}
 requires-python = ">=3.9,<3.14"
