From 42e894ff04867269df8b33a82a1bd673461f1b48 Mon Sep 17 00:00:00 2001
From: Serhii Kupriienko
Date: Thu, 23 Oct 2025 11:05:08 +0300
Subject: [PATCH] fix license

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 3264dce..f179eef 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -25,7 +25,7 @@
 
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools>=77", "Cython", "numpy", "mkl-devel"]
+requires = ["setuptools>=64", "Cython", "numpy", "mkl-devel"]
 
 [project]
 authors = [
@@ -54,7 +54,7 @@ dependencies = ["numpy>=1.26.4", "mkl-service"]
 description = "MKL-based FFT transforms for NumPy arrays"
 dynamic = ["version"]
 keywords = ["DFTI", "FFT", "Fourier", "MKL"]
-license = "BSD-3-Clause"
+license = {text = "BSD"}
 name = "mkl_fft"
 readme = {file = "README.md", content-type = "text/markdown"}
 requires-python = ">=3.9,<3.14"
-- 
2.50.1 (Apple Git-155)

