From 2f0a541c87fbaf30e4c88b0c7c46988db6ea6ef0 Mon Sep 17 00:00:00 2001
From: Ian Fitchet <ifitchet@anaconda.com>
Date: Wed, 20 Aug 2025 18:15:30 +0100
Subject: [PATCH] enable python 3.14

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 55f31da..0d93848 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -56,7 +56,7 @@ keywords = ["DFTI", "FFT", "Fourier", "MKL"]
 license = "BSD-3-Clause"
 name = "mkl_fft"
 readme = {file = "README.md", content-type = "text/markdown"}
-requires-python = ">=3.9,<3.13"
+requires-python = ">=3.9,<3.15"
 
 [project.optional-dependencies]
 scipy_interface = ["scipy>=1.10"]
-- 
2.45.2

